src/components/Shared/CookieBanner.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+
+class CookieBanner extends React.Component {
+
+	constructor(props) {
```

Comment:
You'll notice the tab indentation for most of the code is 2 spaces.  I think we should keep that fairly consistent. 
8 spaces is a lot.

Reply:
I am using a single tab for indentation. I agree 8 spaces is a lot. Maybe it's something in my vs code settings.